butterfly/QueryLayer/TileQuery.py

```diff
@@ -78,11 +82,5 @@ def source_bounds(self):
 
     @property
     def preload_source(self):
-        loader = self.source_class
-        runtime = self.RUNTIME.IMAGE
-        keywords = loader.preload_source(self)
-        # Set one of the keywords to self
-        runtime.BLOCK.VALUE = keywords[runtime.BLOCK.NAME]
-        return keywords
-
-
+        # Preload the metadata from the source
+        return self.source_class.preload_source(self)
```

Comment:
Nothing about the tile_query needs to be updated when preloading, since we always preload using a dummy tile anyway.